app/src/main/java/org/projectbuendia/client/ui/chart/PatientChartController.java

```diff
@@ -694,7 +695,7 @@ public void onEventMainThread(SubmitXformFailedEvent event) {
                 default:
                     errorMessageResource = R.string.submit_xform_failed_unknown_reason;
             }
-            mUi.showError(errorMessageResource);
+            mUi.showFormSubmissionError(errorMessageResource);
```

Comment:
@dancunningham: Do you think these messages should be maintained ? I've created a new message entry to alert the user that the form was not sent and it will be send later.